hwdb.d/60-sensor.hwdb

```diff
@@ -254,6 +254,10 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnCube:pnI15-TC:*
 # Cube iWork 11 Stylus
 sensor:modalias:acpi:KIOX000A*:dmi:*:svncube:pni8-T:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
+ 
```

Comment:
This line seems to contain unnecessary white space. Please drop it.

Reply:
OK,updated